<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_SNR</class>
 <widget class="QWidget" name="Form_SNR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HW THPS SNR Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" stretch="3,2,2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="8,2,1">
         <item>
          <widget class="QLineEdit" name="lineFolder"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnSelectFolder">
           <property name="text">
            <string>Select Folder</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableCFG">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <column>
          <property name="text">
           <string>Config Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Config Value</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Customer</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>icons/huawei_label.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Pattern</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="2" column="3">
              <widget class="QCheckBox" name="cBoxDotInv">
               <property name="text">
                <string>inv. Dot</string>
               </property>
              </widget>
             </item>
             <item row="3" column="4">
              <widget class="QCheckBox" name="cBoxDouble_dot">
               <property name="text">
                <string>Double Dot</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QCheckBox" name="cBoxDot">
               <property name="text">
                <string>Dot </string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QCheckBox" name="cBoxRed">
               <property name="text">
                <string>Red</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QCheckBox" name="cBoxGreen">
               <property name="text">
                <string>Green</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QCheckBox" name="cBoxDouble_dotInv">
               <property name="text">
                <string>inv. Double Dot</string>
               </property>
              </widget>
             </item>
             <item row="2" column="4">
              <widget class="QCheckBox" name="cBoxCol_GP">
               <property name="text">
                <string>Col G/P</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QCheckBox" name="cBoxVertical">
               <property name="text">
                <string>Vertical</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QCheckBox" name="cBoxWhite">
               <property name="text">
                <string>White</string>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QCheckBox" name="cBoxColInv">
               <property name="text">
                <string>inv.Col</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="cBoxCol_GPInv">
               <property name="text">
                <string>inv. Col G/P</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QCheckBox" name="cBoxRow">
               <property name="text">
                <string>Row</string>
               </property>
              </widget>
             </item>
             <item row="4" column="4">
              <widget class="QCheckBox" name="cBoxHorizontal">
               <property name="text">
                <string>Horizontal</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="cBoxBlack">
               <property name="text">
                <string>Black</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QCheckBox" name="cBoxBlue">
               <property name="text">
                <string>Blue</string>
               </property>
              </widget>
             </item>
             <item row="4" column="3">
              <widget class="QCheckBox" name="cBoxRowInv">
               <property name="text">
                <string>inv. Row</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QCheckBox" name="cBoxCol">
               <property name="text">
                <string>Col</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Additional Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="cBoxPlotNoiseptp">
        <property name="text">
         <string>Plot noise p2p</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="cBoxPlotPosNoise">
        <property name="text">
         <string>Plot Mct positive/negative noise</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="cBoxPlotRms">
        <property name="text">
         <string>Plot RMS noise</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="cBoxPlotSctSignal">
        <property name="text">
         <string>Plot all Sct touch signal</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="cBoxPlotMctAveNoise">
        <property name="text">
         <string>Plot Mct average noise</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="cBoxPlotSctPosNoise">
        <property name="text">
         <string>Plot Sct positve/negative noise</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cBoxLog_grid_rawdata">
        <property name="text">
         <string>log grid rawdata</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="cBoxPlotallTouch">
        <property name="text">
         <string>Plot all Mct touch signal</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="cBoxPlot_p2p_annotated">
        <property name="text">
         <string>Plot noise p2p annotated</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="8,2">
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnReset">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClear">
         <property name="text">
          <string>Clear Browser</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnGenerateSNR">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Generate SNR Report</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons/pic.qrc"/>
 </resources>
 <connections/>
</ui>
